src/App.css

```diff
@@ -2,6 +2,9 @@
   text-align: center;
 }
 
+* {
+  font-family: lato;
```

Comment:
Is this font safe for all browsers?

Reply:
I just wonder if there should be a backup font that can take over incase `lato` doesn't render properly.

Reply:
https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/

We can ask @msajara  which font would she prefer for fallback?

Reply:
@GenesisSelf if this is non-blocking, we can always merge so people can use it then come back to add fallback font